ui/app/templates/components/secret-list/item.hbs

```diff
@@ -4,20 +4,20 @@
   class="list-item-row"
   data-test-secret-link=@item.id
   encode=true
-  queryParams=(secret-query-params @backendModel.type)
+  queryParams=(secret-query-params @backendModel.type @item.type)
 }}
   <div class="columns is-mobile">
     <div class="column is-10">
       <SecretLink
         @mode={{if @item.isFolder "list" "show"}}
         @secret={{@item.id}}
-        @queryParams={{if (eq @backendModel.type "transit") (query-params tab="actions") ""}}
+        @queryParams={{secret-query-params @backendModel.type @item.type asQueryParams=true}}
```

Comment:
If this impacts any of your decisions, you can override this component in `options-for-backend` so we have custom stuff specific to KMSE

Reply:
Gotcha would that be the `listItemPartial` prop then? In this case I noticed that there was already the `secret-query-params` helper created for transit and database backends so I was able to leverage that with minor changes.